go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5
 	github.com/fsnotify/fsnotify v1.6.0
 	github.com/gballet/go-libpcsclite v0.0.0-20190607065134-2772fd86a8ff
-	github.com/gballet/go-verkle v0.1.1-0.20231020124853-d124d1998b1a
+	github.com/gballet/go-verkle v0.1.1-0.20231024193227-1127148c02ea
```

Comment:
Pointing to latest commit in https://github.com/gballet/go-verkle/pull/407

When that PR is merged, I'll update this PR to point to master.